### PR TITLE
Renames mask to live_targets for cross_entropy_loss and binary_cross_entropy_loss.

### PR DESCRIPTION
**Pull request by ruomingp**
Motivation: mask is an ambiguous term and should be avoided. https://twitter.com/ericjang11/status/1739411438148739579

### CODE REVIEW

**Review by markblee — APPROVED** [review]

**Comment by ruomingp** [comment]
Need approval again. Thanks!

**Review by markblee — APPROVED** [review]

File: axlearn/common/loss.py [hunk 1]
```diff
@@ -141,64 +146,80 @@ def cross_entropy(
...
+    if live_targets is None:
+        live_targets = jnp.logical_and(0 <= target_labels, target_labels < num_classes)
+    live_targets = live_targets.astype(per_target_loss.dtype)
+    denominator = jnp.maximum(live_targets.sum(), 1)
+    cross_entropy_loss = (per_target_cross_entropy_loss * live_targets).sum() / denominator
+    z_loss = (per_target_z_loss * live_targets).sum() / denominator
+    loss = (per_target_loss * live_targets).sum() / denominator
     predicted_labels = jnp.argmax(logits, axis=-1)
-    accuracy = (jnp.equal(predicted_labels, target_labels) * mask).sum() / num_unmasked
+    accuracy = (jnp.equal(predicted_labels, target_labels) * live_targets).sum() / denominator
```
Comment (apivovarov) [inline axlearn/common/loss.py:165]: Hi Ruoming, should we use this accuracy inside CrossEntropyLossMetrics::forward() instead of recalculating it in CrossEntropyLossMetrics::forward() ?
see: https://github.com/apple/axlearn/blob/main/axlearn/common/causal_lm.py#L108C1-L117C10
@ruomingp
